package.yaml

```diff
@@ -1,6 +1,6 @@
 ---
 name: yesod-markdown
-version: '0.12.1'
+version: '0.12.2'
```

Comment:
Please avoid the version bump in the PR. I'll do that myself (along with other things) between merge and release.

Reply:
Sure thing!